index.html

```diff
@@ -1226,6 +1247,55 @@
       <a href="#steps-receiving">Receiving and parsing content</a>
       and <a>Writing or pushing content</a> sections.
     </p>
+    <p>
+      To <dfn>convert NDEFRecord's bytes</dfn>, pass a <var data-type="NDEFRecord">record</var>
+      and a <var>type</var>, run these steps:
+    </p>
+    <ol>
+      <li>
+        Let <var>bytes</var> be the <var>record</var>'s <a>bytes</a>.
+      </li>
+      <li>
+        Let <var data-type="NDEFRecordType">recordType</var> be the value of <var>record</var>'s
+        recordType attribute.
+      </li>
+      <li>Switch on <var>type</var>:
+        <dl>
+          <dt>text</dt>
+          <ol>
+            <li>
+              If the <var>recordType</var> value is not equal to <code>"empty"</code>, then
+              return the result of running <a>UTF-8 decode</a> on <var>bytes</var>.
+            </li>
+            <li>
+              Otherwise, return `null`.
+            </li>
+          </ol>
+          <dt>arrayBuffer</dt>
+          <ol>
+            <li>
+              If the <var>recordType</var> value is equal to <code>"json"</code> or
+              <code>"opaque"</code>, then return an {{ArrayBuffer}} whose contents are the
```

Comment:
This is how we write these in the Wakelock spec using latest respec

```
"`wake-lock`"
```

Reply:
Done.